packages/cactus-plugin-ledger-connector-cdl-socketio/tsconfig.json

```diff
@@ -0,0 +1,27 @@
+{
+  "extends": "../../tsconfig.base.json",
+  "compilerOptions": {
+    "rootDir": "./src/main/typescript",
+    "composite": true,
+    "outDir": "./dist",
+    "sourceMap": false,
+    "emitDecoratorMetadata": true,
+    "experimentalDecorators": true,
+    "tsBuildInfoFile": "../../.build-cache/cactus-plugin-ledger-connector-cdl-socketio.tsbuildinfo"
+  },
```

Comment:
@outSH Is there a specific reason why this cannot be like the rest of the packages (like the common package's tsconfig) when it comes to configuring the output directories (outDir, declarationDir, etc.)?

If it's all the same to you I'd refactor it to match to reduce future risk of build problems that arise from not having uniform directory structures.
We already have other packages where the same problem exists so it's not a big deal to make it happen here either, but I wanted to bring it up for at least future reference.

```json
{
  "extends": "../../tsconfig.base.json",
  "compilerOptions": {
    "composite": true,
    "outDir": "./dist/lib/",
    "declarationDir": "dist/lib",
    "rootDir": "./src",
    "tsBuildInfoFile": "../../.build-cache/cactus-common.tsbuildinfo"
  },
  "include": [
    "./src"
  ],
  "references": []
}
```

Reply:
@petermetz Of course - fixed